### ISSUE TITLE
cref seems to be incompatible

### ISSUE THREAD

**Issue by JonasOberhauser** [yforster/coqtheorem#4]
Not sure why, but it appears cleverref doesn't know the labels of normal environments anymore. Probably you guys are nesting environments or something?

**Comment by yforster** [comment]
You're right, I don't remember why it was there in the first place. Thanks!

**Comment by yforster** [comment]
Strange, which environments stop working for you? Linking to figures still works, e.g. copy pasting this into `example-lipics.tex` produces correct results:

```
\begin{figure}
  Bla
  \caption{Test}
  \label{fig:test}
\end{figure}

Test~\Cref{fig:test}
```
Do you load LIPIcs with the `cleveref` option, e.g. `\documentclass[cleveref]{lipics-v2019}`?

**Comment by JonasOberhauser** [comment]
For example, lemmas seem not to work anymore (here's to hoping I didn't just forget to recompile a few times after changing to your package 💃 ). 

Here is my preamble:
```

\documentclass[a4paper,UKenglish,cleveref, autoref]{lipics-v2019}
%This is a template for producing LIPIcs articles. 
%See lipics-manual.pdf for further information.
%for A4 paper format use option "a4paper", for US-letter use option "letterpaper"
%for british hyphenation rules use option "UKenglish", for american hyphenation rules use option "USenglish"
%for section-numbered lemmas etc., use "numberwithinsect"
%for enabling cleveref support, use "cleveref"
%for enabling cleveref support, use "autoref"

\usepackage{import}

%	\kern0.05em{\color[rgb]{0.99,0.78,0.07}\rule{0.73em}{0.73em}}%
%	\hspace*{0.67em}\bothIfFirst{#1}{~}#2}
\newaliascnt{listing}{table}
\newaliascnt{statement}{equation}
\crefname{program}{Listing}{Listings}
\Crefname{program}{Listings}{Listings}

%% We use gather etc. not for equations but for statements
\crefname{statement}{Statement}{Statements}
\Crefname{statement}{Statement}{Statements}


\lstnewenvironment{tabularlstlisting}[1][]
{%
	\lstset{aboveskip=-1.3ex,belowskip=-3.5ex,#1}%
}
{}

```

**Comment by yforster** [comment]
Mh, if I add

```
\begin{document}

\begin{lemma}\label{test}
  Bla
\end{lemma}

\Cref{test}

\end{document}
```
after your preamble the `\Cref` correctly prints `Lemma 1`. This might still be a bug, but maybe indeed try recompiling. I'm using `lipics-v2019 2018/12/10 v2.0 LIPIcs articles` (it's in one of the first few lines of the compilation log) btw

**Comment by JonasOberhauser** [comment]
I tried recompiling, it still shows me this:
![image](https://user-images.githubusercontent.com/183564/55292302-3d279380-53e9-11e9-828b-9bb4a0b07b2e.png)

The interesting thing is that it sometimes works, for example "proof of theorem..." is generated with cref, but directly afterwards it does not understand it anymore. A few pages later, it works again (also with lemma):
![image](https://user-images.githubusercontent.com/183564/55292341-c17a1680-53e9-11e9-8ef5-cb62de6703d4.png)


I'm using
```
\ProvidesClass{lipics-v2019}
    [2018/12/10 v2.0 LIPIcs articles]
```

**Comment by JonasOberhauser** [comment]
That it works in some cases but not in others hints at something wrong with my file, I'll see if there's something strange I'm doing.

**Comment by JonasOberhauser** [comment]
I think I figured out what the problem relates to. It happens if multiple items are of the same type in the cref, like
\cref{lemma1,lemma2}. 

You're probably not defining the plurals correctly...
